src/main/g8/src/main/scala/$package__packaged$/Server.scala

```diff
@@ -1,15 +1,22 @@
 package $package$
 
-import fs2.{Task, Stream}
-import org.http4s.util.StreamApp
-import org.http4s.server.Server
+import java.util.concurrent.{ExecutorService, Executors}
+
+import fs2.Task
+
+import scala.util.Properties.envOrNone
 import org.http4s.server.blaze.BlazeBuilder
-import scala.concurrent.ExecutionContext
-
-object BlazeExample extends StreamApp {
-  override def stream(args: List[String]): Stream[Task, Nothing] =
-    BlazeBuilder
-      .bindHttp(8080, "0.0.0.0")
-      .mountService(HelloWorld.service)
-      .serve
-}
+import org.http4s.util.StreamApp
+
+
+object Server extends StreamApp {
+
+  val port : Int              = envOrNone("HTTP_PORT") map (_.toInt) getOrElse 8080
+  val ip   : String           = "0.0.0.0"
+  val pool : ExecutorService  = Executors.newCachedThreadPool()
```

Comment:
This doesn't get used, does it?

I also don't recommend cached thread pools on a server, because they grow without bound.

Reply:
your right, I had that executor from older testing and copied it in by mistake